homeassistant/components/media_player/samsungtv.py

```diff
@@ -104,7 +105,7 @@ def __init__(self, host, port, name, timeout, mac, uuid):
         self._muted = False
         # Assume that the TV is in Play mode
         self._playing = True
-        self._state = None
+        self._state = STATE_UNKNOWN
```

Comment:
Keep this as `None`, we should never use `STATE_UNKNOWN` as that will be dealt with automatically in the entity helper class.